ReactNativeClient/android/app/src/main/java/net/cozic/joplin/MainApplication.java

```diff
@@ -51,7 +53,12 @@ public ReactNativeHost getReactNativeHost() {
 	public void onCreate() {
 		super.onCreate();
 
-		// To try to fix the error "Row too big to fit into CursorWindow"
+        // Enable debugging with the WebView we use to display notes
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            WebView.setWebContentsDebuggingEnabled(true);
```

Comment:
tabs

Reply:
I noticed that, but git and android studio told me there isn't a tab error
![image](https://user-images.githubusercontent.com/33200024/75416356-1ef94900-5954-11ea-800e-7d5be4102d01.png)

Very strange stuff. If I check the same file on VSCode, it shows there are two extra tabs!
![image](https://user-images.githubusercontent.com/33200024/75416374-2ddffb80-5954-11ea-96ae-ea36d016ac89.png)

First I thought it was a tabs vs spaces error. Tried reseting to master and making the changes again with tabs. Same result! Any idea how to fix?

Reply:
Ah. Found the error. So throughout that file, there is no consistent use of spaces vs tabs. Because of that, both my editors get confused as to what do display and how to edit the file (whether it should enter spaces when I press tab) and how many spaces to use when inserting spaces. Fixed though.